pkg/kubectl/cmd/util/cmdtools.go

```diff
@@ -190,3 +235,12 @@ func platformize(linux, windows string) string {
 	}
 	return linux
 }
+
+func checkGitDiffInstalled(gitdiff, diff string) string {
+	cmd := exec.Command("git", "version")
```

Comment:
Is using `hash git` make for sense here? But the question is does `hash` works on windows?